presto-main/src/main/java/io/prestosql/server/protocol/Query.java

```diff
@@ -253,6 +256,11 @@ public QueryId getQueryId()
         return queryId;
     }
 
+    public String getSlug()
```

Comment:
Instead of adding a getter, maybe add `boolean isSlugValid(String slug)`

Reply:
good idea